controllers/historyController.js

```diff
@@ -52,6 +52,24 @@ const getHistoryInfo = async ( req, res, next ) => {
   }
 }
 
+const findHistoryUserEntry = async ( req, res, next ) => {
+  try {
+    const userId = req.user.id;
+    const sudokuId = req.params.sudokuId;
+    if ( await getHistoryByUserIdSudokuId( userId, sudokuId ) ) {
+      res.status(200).send({
+        payload: { hasSolved: true }
```

Comment:
hasSolved could be interpreted as already completed sudoku. maybe 'hasHistoryEntry' would be more descriptive.